src/quo2/components/input/view.cljs

```diff
@@ -0,0 +1,173 @@
+(ns quo2.components.input.view
+  (:require [oops.core :as oops]
+            [quo2.components.icon :as icon]
+            [quo2.components.input.style :as style]
+            [quo2.components.markdown.text :as text]
+            [react-native.core :as rn]
+            [reagent.core :as reagent]))
+
+(defn- label-&-counter
```

Comment:
is it legal? :) but looks kinda strange, why not `and` ?

Reply:
Yes, it works, I didn't notice it was strange. I'll replace that with `and` on the following test adding PR.

As a side note, some years ago I had a job where I was asked to write Clojure in Spanish, so Clojure supports all of its special characters (such as áéíóúüñ).